processor/README.md

```diff
@@ -22,25 +22,28 @@ The [contrib repository](https://github.com/open-telemetry/opentelemetry-collect
 ## Recommended Processors
 
 By default, no processors are enabled. Depending on the data source, it may be recommended that multiple processors be enabled. Processors must be
-enabled for every data source: Not all processors support all data sources.
+enabled for every data source: not all processors support all data sources.
 In addition, it is important to note that the order of processors matters. The
 order in each section below is the best practice. Refer to the individual
 processor documentation for more information.
 
 ### Traces
 
 1. [memory_limiter](memorylimiterprocessor/README.md)
-2. *any sampling processors*
+2. *Sampling processors, excluding `tail_sampling`*
```

Comment:
Not sure I understand this, can you explain to me?

Reply:
`tail_sampling` is the name of a sampling processor, however if it is added at this point, it will break `k8sattributes`. This line is saying "Add sampling processors, apart from `tail_sampling`". `tail_sampling` needs to be added below. It sounds like it may need a reword - how can that be made clearer?